ci/ansible/roles/pulp-crane/files/pulp_crane_el6.conf

```diff
@@ -0,0 +1,16 @@
+# source:
+# https://raw.githubusercontent.com/pulp/crane/3c657fe53020b804ce60d907a82b4fb19aebe532/deployment/apache22.conf
+# Use this config with Apache 2.2 or earlier
+Listen 5000
+<VirtualHost *:5000>
+    WSGIScriptAlias / /usr/share/crane/crane_el6.wsgi
+    <Location /crane>
+        Order deny,allow
+        Deny from all
+        Allow from localhost
```

Comment:
The change I suggested above works. You can test for yourself by executing this on the Pulp+Crane system:

```sh
curl -H accept:application/json http://localhost:5000/crane/repositories
```

...and this from some other host:

```sh
curl -H accept:application/json http://${hostname}:5000/crane/repositories
```

Before the change, the latter request results in an HTTP 403 "forbidden" response. After the change, the request is answered.